src/addons/transitions/__tests__/ReactCSSTransitionGroup-test.js

```diff
@@ -132,29 +128,28 @@ describe('ReactCSSTransitionGroup', () => {
   it('should switch transitionLeave from false to true', () => {
     var a = ReactDOM.render(
       <ReactCSSTransitionGroup
-          transitionName="yolo"
-          transitionEnter={false}
-          transitionLeave={false}>
+        transitionName="yolo"
+        transitionEnter={false}
+        transitionLeave={false}
```

Comment:
`--jsx-bracket-same-line` (btw, all the options have been added to match the fb styleguide, so you want to put them all :p)